lib/registration.pm

```diff
@@ -428,6 +423,7 @@ sub fill_in_registration_data {
                 assert_screen 'yast-scc-emptypkg';
                 send_key 'alt-a';
             }
+            accept_addons_license('ha') if (check_var('SLE_PRODUCT', 'sles4sap'));
```

Comment:
I'm adding this line for the special case that is SLES4SAP (product includes the HA extension automatically without the need to add it with `SCC_ADDONS`), but figure there could be other cases (perhaps HPC?) which would require a more thorough code to be put in its place.

@asmorodskyi @rwx788 @okurz @ldevulder Thoughts?

Reply:
I would vote for merge it like this and when other addons ( including HPC)  will face with this issue we will think about changing this to more generic approach. Let's not over-engineer and solve problems when they come.